ui/index.ts

```diff
@@ -1,8 +1,19 @@
 import App from "./App.svelte";
 
+const parseBackendAddress = (): string | undefined => {
+  const search = window.location.search;
+
+  if (search.includes("?backend=")) {
+    const match = search.match(/\?backend=(.*)/);
```

Comment:
An alternative would be to use [`qs`](https://www.npmjs.com/package/qs)

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/4420975ab7cfc41d091792d380503bc29ab62e4b